projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -749,6 +937,41 @@ async function main(_node, _script, ...args) {
 		}
 	}
 
+	try {
+		await go(options);
+	}
+	finally {
+		if (readline) {
+			readline.close();
+		}
```

Comment:
If we extract `prompt()`, we'll also extract a clean-up function into the same place, to be called from here.